app/src/lib/databases/pull-request-database.ts

```diff
@@ -23,6 +23,9 @@ export interface IPullRequest {
   /** The title. */
   readonly title: string
 
+  /** The body of the PR - This is markdown. */
```

Comment:
Sorry, this came to mind while reading your comment, I couldn't help it 😳 🤣 

![5xopk7](https://user-images.githubusercontent.com/1083228/145805411-77d3325b-5dc4-4aa2-9170-32a8c6e709d3.gif)

Reply:
😂🤣😂🤣👏👏👏👍🥰🌞✊🙏🇺🇸🌎🌎🤪😁